openrouteservice/src/main/java/org/heigit/ors/services/matrix/requestprocessors/json/JsonMatrixRequestProcessor.java

```diff
@@ -13,24 +13,14 @@
  */
 package org.heigit.ors.services.matrix.requestprocessors.json;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import org.json.JSONArray;
-import org.json.JSONObject;
-
 import com.graphhopper.util.Helper;
 import com.vividsolutions.jts.geom.Coordinate;
-
+import org.heigit.ors.api.util.SystemMessage;
 import org.heigit.ors.common.StatusCode;
```

Comment:
As said above, a core class should not import an API class.

Reply:
that's old deprecated v1 api response production that should have been in the api package only it didn't exist I guess. Wouldn't care fixing code that's deprecated anyway.

Reply:
ditto.